Automattic/woocommerce-payments
# Fix hover dialog for close button on modals, unify styling and layout of modal buttons.

Closes #1729

@LevinMedia I know we _just_ removed the close button from the modal per our conversation, but I believe adding it back would be best here to be in line with the modals in WooCommerce core.

#### Changes proposed in this Pull Request

* Hide the dialog that pops up when hovering over the close X in modals, as described in #1729.
* Updated modal buttons to all have the same layout where the cancel and actions buttons are on the right side, with the action button being in the lower right corner. This matches default WordPress and WooCommerce layout.
* Created mixin that can be used to style all modal buttons.

#### Testing instructions

1. In WCPay Dev tools, _ Enable grouped settings_.
2. Go to Payments > Settings, then click on _Add payment method_ in the first section.
3. Modal will open up, hover over the X, no scrollbars should appear.
4. Confirm the buttons are on the lower right, with _Add selected_ in the corner and _Cancel_ to the left of it.

-------------------


Default WooCommerce modal:
![Screenshot on 2021-05-07 at 10-02-40](https://user-images.githubusercontent.com/4634416/117464082-b417aa80-af26-11eb-9686-973146a3b31c.png)

Modal mentioned in testing above:
![Screen Shot 2021-05-07 at 11 24 18 AM](https://user-images.githubusercontent.com/4634416/117464203-cf82b580-af26-11eb-9bd0-7a3d1568ab9f.png)
